src/components/Map/map.js

```diff
@@ -265,6 +265,28 @@ class Map extends Component {
     });
   };
 
+  addHoverHandlerToMapIdLayer = typeId => {
+    let popup = new mapboxgl.Popup({
+      closeButton: false,
+      closeOnClick: false,
+      className: "name-popup",
+      offset: {'bottom': [0, -20]},
```

Comment:
If the icon is near the edge of the map, mapbox seems to move the popup to keep it in frame, and the popup overlaps the icon if not on the bottom. I think we can replace  {'bottom': [0, -20]} with 20 to have mapbox maintain that distance on all sides

Reply:
nice! thanks